tests/cpptests/test_cpp_forkgc.cpp

```diff
@@ -14,6 +14,25 @@ extern "C" {
 
 #include <set>
 
+/**
```

Comment:
Nice work on the tests - which ones would have failed without the changes in this PR?

Reply:
I don't have an option to reply to your previous comment for some reason so I'll do it here - 
`ixmsg.nbytesCollected` contains the summary of bytes collected from all the blocks in the inverted index,
while `ixmsg.lastblkBytesCollected` should contain only the last block's bytes count, in case the parent process would want to discard the changes made to the last block. In this case, when the parent discards the changes of the last block, it needs to update the general statistics (total bytes collected), by decreasing the last block's byes count from it. So before the fix, the general statistics of how many bytes were cleaned became zero and the inverted index size field constantly increased.

Regarding the tests - I checked in most tests that the number of entries and the inverted index size [mb] is updated as expected when skipping the last block.

Before this fix `ixmsg.lastblkBytesCollected` was eventually equal to the total bytes collected and if the last block was skipped,